src/python/WMCore/MicroService/Unified/MSOutput.py

```diff
@@ -503,17 +518,21 @@ def msOutputProducer(self, requestRecords):
         #    to set a destructive function at the end of the pipeline
         # NOTE:
         #    To discuss the collection names
+        # NOTE:
```

Comment:
I don't get this comment. You do add self.docUploader to the pipeline here. What am I missing?

Reply:
The comment is not regarding the `docUploader()` function itself. It is why it is added with the default value of the `update=False` parameter. It is just a reminder, because I was quite close earlier this morning to fool myself  and just put `update=True` here so that the newly generated `destinationMap` can be updated without the need for deleting the whole MongoDB during deployment time. But this would break the logic for the Consumer thread then.